README.md

```diff
@@ -156,6 +156,21 @@ These Options are available to customize the installation:
 
 **SORMAS_S2S_REST_PASSWORD** Password for the rest user to connect to sormas
 
+### PIA
+If you choose to use the align SORMAS with a PIA instance, use the docker-compose-sb.yml.
```

Comment:
If you choose to ~~use the~~ align SORMAS with a PIA instance, use the docker-compose-sb.yml.